simulation/traffic_simulator/src/api/api.cpp

```diff
@@ -397,4 +425,52 @@ void API::requestLaneChange(
   entity_manager_ptr_->requestLaneChange(name, target, trajectory_shape, constraint);
 }
 
+void API::requestSpeedChange(const std::string & name, double target_speed, bool continuous)
+{
+  if (entity_manager_ptr_ == nullptr) {
+    THROW_SIMULATION_ERROR(__FILE__, __LINE__, "Entity manager is not initialized");
```

Comment:
THROW_SIMULATION_ERROR macro includes file and line infomation.

```suggestion
    THROW_SIMULATION_ERROR("Entity manager is not initialized");
```

Reply:
Also, entity_manager_ptr_ is initialized in member initialized list, so I think this error means the simulator was broken.
So please leave message such as "If you see this error, something completely unexpected happens. Please contact developer."